examples/llm/aiq_baseline/langchain_corpus_ingest.py

```diff
@@ -0,0 +1,260 @@
+            messages=[
+                {
+                    "role": "system",
+                    "content": SYSTEM_PROMPT
+                },
+                {
+                    "role": "user",
+                    "content": USER_PROMPT
+                },
+            ],
+            #temperature=0.2,
+            #top_p=0.7,
+            #max_tokens=1024,
+            #stream=True
+        )
+
+        # FIXME: Trim response that will be passed to LLM
+        print(response.choices[0].message.content)
+        response = response.choices[0].message.content
+
+        # NOTE: update the env variables for the 'NV_NIM_KEY' and the 'ENDPOINT_URL'
+        llm_judge = LLMJudge()
+        score.append(llm_judge.score(question, response, answer))
+
+
+def emb_text(text_lines, model):
+    embeddings = model.encode(text_lines)
+    return embeddings
+
+
+def main(*, milvus_uri: str, collection_name: str, dataset: str,
+         llm_generator_name: str, drop_collection: bool, embedding_model: str,
+         chunk_size: int, chunk_overlap: int, metric_type: str,
+         with_react_agent: bool):
+
+    dir_to_read = os.path.join(dataset, "corpus")
+    files_to_read = os.path.join(dir_to_read, "*.txt")
+    text_lines = []
+    logger.info("Divide large bodies of text into smaller chunks")
+    for file_path in glob(files_to_read, recursive=True):
+        with open(file_path) as file:
+            file_text = file.read()
+        text_splitter = RecursiveCharacterTextSplitter(
+            chunk_size=chunk_size,  # Example chunk size
+            chunk_overlap=chunk_overlap,  # Example chunk overlap
+            # separators=["\n\n", "\n", " ", ""], # Default separators
+            length_function=len,  # Function to measure chunk length
+            is_separator_regex=False  # Whether separators are regex patterns
+        )
+
+        chunks = text_splitter.split_text(file_text)
+        text_lines += chunks
+
+    device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+    encoder_model = SentenceTransformer(
+        model_name=embedding_model).to(device).eval()
+
+    test_embedding = emb_text("This is a test", encoder_model)
+    # The test above is used to retrieve the embedding dim
+    embedding_dim = len(test_embedding[0])
+
+    # Setting the uri as a local file, e.g../milvus.db, is the most convenient method,
+    # as it automatically utilizes Milvus Lite to store all data in this file.
+
+    # If you have large scale of data, you can set up a more performant Milvus server on
+    # docker or kubernetes. In this setup, please use the server uri, e.g.http://localhost:19530, as your uri.
+    # If you want to use Zilliz Cloud, the fully managed cloud service for Milvus, adjust the uri and token,
+    # which correspond to the Public Endpoint and Api key in Zilliz Cloud.
+    milvus_client = MilvusClient(uri=milvus_uri)
+
+    # FIXME: For now, always drop collection prior to each run
+    if drop_collection:
```

Comment:
Is this `FIXME` still accurate? It looks like `drop_collection` is passed in from the CLI and may not always be `True`.